src/parser/jobs/rpr/modules/ImmortalSacrifice.tsx

```diff
@@ -0,0 +1,154 @@
+import {t} from '@lingui/macro'
+import {Trans} from '@lingui/react'
+import React from 'react'
+import {DataLink} from '../../../../components/ui/DbLink'
+import {Event, Events} from '../../../../event'
+import {Analyser} from '../../../core/Analyser'
+import {filter} from '../../../core/filter'
+import {dependency} from '../../../core/Injectable'
+import BrokenLog from '../../../core/modules/BrokenLog'
+import Checklist, {Requirement, Rule} from '../../../core/modules/Checklist'
+import {Data} from '../../../core/modules/Data'
+import {DISPLAY_ORDER} from './DISPLAY_ORDER'
+
+interface ImmortalSacrificeWindow {
+	startTime: number,
+	endTime?: number,
+	/**
+	 * If this window was closed, by consuming the stacks, or it expired
+	 */
+	closed: boolean,
+	stackCount: number,
+	consumed: boolean,
+}
+
+export default class ImmortalSacrifice extends Analyser {
+
+	static override handle = 'immortalsacrifice'
+	static override title = t('rpr.immortal-sacrifice.title')`Immortal Sacrifice stacks consumed`
```

Comment:
The title is for the navigation/panel, just `Immortal Sacrifice` is fine. I'd maybe rethink the i18n module name too so it's at least consistent with the module handle.

Reply:
changed the handle name